packages/contracts/contracts/libs/v0.8.x/ERC721_PackedHashSeedV1.sol

```diff
@@ -0,0 +1,592 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v5.0.0) (token/ERC721/ERC721.sol)
+
+pragma solidity ^0.8.20;
+
+import "@openzeppelin-5.0/contracts/token/ERC721/IERC721.sol";
+import "@openzeppelin-5.0/contracts/token/ERC721/IERC721Receiver.sol";
+import "@openzeppelin-5.0/contracts/token/ERC721/extensions/IERC721Metadata.sol";
+import "@openzeppelin-5.0/contracts/utils/Address.sol";
+import "@openzeppelin-5.0/contracts/utils/Context.sol";
+import "@openzeppelin-5.0/contracts/utils/Strings.sol";
+import "@openzeppelin-5.0/contracts/utils/introspection/ERC165.sol";
+import "@openzeppelin-5.0/contracts/interfaces/draft-IERC6093.sol";
```

Comment:
can you update this to use named imports, same as OpenZeppelin's implementation?

Reply:
Good call, updated!